WalletWasabi.Gui/Controls/ExtendedTextBox.cs

```diff
@@ -128,11 +128,9 @@ private async Task CopyAsync()
 
 		private static DrawingPresenter GetCopyPresenter()
 		{
-			return new DrawingPresenter
-			{
-				Drawing = new GeometryDrawing
-				{
-					Brush = Brushes.LightGray,
+			return new DrawingPresenter {
+				Drawing = new GeometryDrawing {
+					Brush = Brush.Parse("#22B14C"),
```

Comment:
Agree. Opened an issue for that. Nevertheless, this is unrelated to the PR, since it must be done in the Shell: https://github.com/zkSNACKs/WalletWasabi/issues/1394